model_compression_toolkit/gptq/keras/quantizer/gumbel_rounding/symmetric_gumbel.py

```diff
@@ -141,8 +141,8 @@ def build(self,
                                           signed=True,
                                           power_of_two=self.power_of_two)
 
-        ceil_indicator = (q_error < 0).numpy().astype("int")  # Negative error means the choose point is celled
-        auxvar_tensor.assign(init_aux_var(ceil_indicator, self.w_shape, self.m))
+        celled_indicator = (q_error < 0).numpy().astype("int")  # Negative error means the choose point is celled
```

Comment:
I meant that the word "celled" in the comment has a typo, not the variable name (which was OK)